spyderlib/config.py

```diff
@@ -512,17 +514,45 @@ def is_ubuntu():
               'editor/toggle comment': "Ctrl+1",
               'editor/blockcomment': "Ctrl+4",
               'editor/unblockcomment': "Ctrl+5",
+              'editor/start of line': "Meta+A",
+              'editor/end of line': "Meta+E",
+              'editor/previous line': "Meta+P",
+              'editor/next line': "Meta+N",
+              'editor/previous char': "Meta+B",
+              'editor/next char': "Meta+F",
+              'editor/previous word': "Meta+Left",
+              'editor/next word': "Meta+Right",
+              'editor/kill to line end': "Meta+K",
+              'editor/kill to line start': "Meta+U",

```

Comment:
Gnome3 uses the Super/Windows/Meta key a lot too. For the record I use int in every DE I use because it's usually not used in programs (contrary to Alt for example).
As these seem mostly useful in some specific presets only, maybe there could be no default keybind, but the user could add them as he like ? I mean, to go to end of line I just use the `End` key...